hack/.golint_failures

```diff
@@ -9,7 +9,6 @@ cmd/kubeadm/app/apis/kubeadm/v1alpha1
 cmd/kubelet/app
 cmd/kubelet/app/options
 cmd/kubemark
-examples/guestbook-go
```

Comment:
Please remove this change.

Reply:
Removing change. Just an quick question,
1. Why golint reports error for these packages while running '`make verify`, even if they are added into `.golint_failures` list? 
2. Should we keep ignoring linting errors if entries found in `.golint_failures`?

Reply:
@cblecker should we remove `examples/guestbook-go` from `.golint_failures`, as `examples/` has been removed from upstream ?